src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-link {
+  text-align: center;
+}
+
+a.is-active {
+  color: #00acdc;
+  text-decoration: underline;
+  position: relative;
+}
+
+.is-active::after {
+  content:"";
+  position: absolute;
+  top: 30px;
+  left: 0;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
```

Comment:
It seems that the `::after` pseudo-element of the `.is-active` class is not properly centered. The `left` property is set to `0`, which means the element will start at the very left of the `.is-active` element. You might want to adjust this to ensure the element is centered correctly.